rbac/management/role/serializer.py

```diff
@@ -99,7 +99,7 @@ class Meta:
 
     def get_applications(self, obj):
         """Get the list of applications in the role."""
-        obtain_applications(obj)
```

Comment:
I feel like some sort of code analysis tooling could have called attention to this, but the fix is pretty simple so I'll give it a 👍